### ISSUE TITLE
Nett wäre eine Config-Zeile für Auflösung

### ISSUE THREAD

**Issue by KarlZeilhofer** [ElektronikNode/scicalc#4]
Für technische Berechnungen sind of 3 Ziffern ausreichen, bei Buchhaltung braucht es aber oft 5 oder 6 Stellen. 
Das wäre super, wenn das über eine Config-Zeile eingestellt werden könnte. 

Man könnte sich von Doxygen den Kommentarstil abkupfern:
```
/// digits=5
```

**Comment by feichtinger** [comment]
Ja, sowas wäre überlegenswert. Dann stellt sich natürlich die Frage ob man das nicht gleich auf weitere Einstellungen erweitern soll.
Ich würde eher dazu tendieren diese Einstellungen in Form einer Funktion vorzunehmen zb. so:

setDigits(5);

Ansonsten besteht die Gefahr dass wir auf kurz oder lang einen 2. Parser für den Kommentarteil schreiben.

**Comment by KarlZeilhofer** [comment]
Es gibt nun 
* setDigits(n) mit n=1...15
* setAccounting(b) und 
* setTrailingZeros(b) mit b=0 oder b=1

implementiert in cd582363bfe4742377807bb08b1e2b10b21bce49

<img width="742" height="462" alt="Image" src="https://github.com/user-attachments/assets/dbc7ca22-2fc1-405c-a71f-8ff9bd464406" />
